API/common/utils.py

```diff
@@ -200,6 +200,12 @@ def exists(path):
     '''
     return os.path.exists(path)
 
+def exist_files(path, files):
+    for file in files:
```

Comment:
You should use an other variable name for `file`. E.g. filename. `file` is a Python built-in keyword.